priv/repo/migrations/20240228055631_update_enrollment_record_table.exs

```diff
@@ -0,0 +1,12 @@
+defmodule Dbservice.Repo.Migrations.UpdateEnrollmentRecordTable do
+  use Ecto.Migration
+
+  def change do
+    alter table(:enrollment_record) do
+      remove :date_of_school_enrollment
+      remove :date_of_group_enrollment
+      add :date_of_enrollment, :date
+      add :group_type, :string
```

Comment:
We should add `group_id` here also, right?

Reply:
actually there already is a group_id in the table. But I do need to rename it.